src/components/KBreadcrumbs/KBreadcrumbs.vue

```diff
@@ -113,51 +125,56 @@ export default {
     .k-breadcrumb-divider,
     .k-breadcrumb-icon {
       align-self: center;
-      color: var(--grey-500, color(grey-500));
+      color: $kui-color-text-decorative;
       display: inline-flex;
     }
 
     .k-breadcrumb-divider {
-      padding: 0 var(--spacing-sm, spacing(sm)) 0 var(--spacing-xs, spacing(xs));
+      padding: 0 $kui-space-20;
```

Comment:
Another token example
```suggestion
      padding: var(--kui-space-0, $kui-space-0) var(--kui-space-20, $kui-space-20);
```